datalad_next/iterable_subprocess/test_iterable_subprocess.py

```diff
@@ -128,12 +128,13 @@ def test_exception_from_not_found_process_propagated():
 
 
 def test_exception_from_return_code():
-    with pytest.raises(IterableSubprocessError, match='No such file or directory') as excinfo:
+    with pytest.raises(IterableSubprocessError) as excinfo:
         with iterable_subprocess(['ls', 'does-not-exist'], ()) as output:
             a = b''.join(output)
 
     assert excinfo.value.returncode > 0
-    assert b'No such file or directory' in excinfo.value.stderr
+    assert (b'No such file or directory' in excinfo.value.stderr) \
+           or (b'Datei oder Verzeichnis nicht gefunden' in excinfo.value.stderr)
```

Comment:
Let's not do this. Instead, patch the ENV to set `LANG=C`. This is a safe pattern. The solution above is waiting for 195 other locales to be added.

In general, this is hinting at a possible extension of `iter_subproc()` (not for this test, though), to accept ENV overrides. Parsing of program output is always fragile, and should only happen as a last resort. We would postponed such an addition to a later point in time.